### PR TITLE
add-list-of-books

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary

I added list of books.

## Screenshots

gif

## Trello Card

https://trello.com/c/MAbYvJNA/30-listado-de-libros

### CODE REVIEW

File: maquetado/scss/detail.scss [hunk 1]
```diff
@@ -2,85 +2,42 @@
 
 .book-title {
   font-family: Roboto;
-  font-size: 30px;
+  font-size: 16px;
```
Comment (gabrielizalo) [inline maquetado/scss/detail.scss]: Can it be `15px`?

File: maquetado/scss/detail.scss [hunk 2]
```diff
@@ -2,85 +2,42 @@
...
-  &::after {
-    background-color: $green;
-    content: "";
-    height: 5px;
-    margin-top: 20px;
-    position: absolute;
-    width: 640px;
-    top: calc(70%);
-  }
+  line-height: 18px;
```
Comment (gabrielizalo) [inline maquetado/scss/detail.scss]: Can it be `20px`?

File: maquetado/scss/detail.scss [hunk 3]
```diff
@@ -2,85 +2,42 @@
...
-  margin-top: 10px;
-  max-width: 1000px;
-  padding: 10px;
-  position: relative;
+  flex-direction: column;
+  height: 385px;
+  max-width: 265px;
+  padding: 30px 30px 50px 30px;
   width: 100%;
+  margin: 30px;
```
Comment (gabrielizalo) [inline maquetado/scss/detail.scss]: Move `margin` before `max-width`.

**Review by gabrielizalo — CHANGES_REQUESTED** [review]

File: maquetado/index.html [hunk 4]
```diff
@@ -8,27 +8,79 @@
...
+    <ol class="list-of-books">
+      <li>
+        <div class="book-cover">
+          <img class="book-image" src="image/book-cover.png" alt="Book Image" />
+          <div class="book-detail">
+            <h3 class="book-title">Título del libro</h3>
+            <p class="book-subtitle">Autor del libro</p>
+          </div>
+        </div>
+      </li>
```
Comment (Dorian30) [inline maquetado/index.html]: There is no need to use ordered or ordered list for this. You usually use these when it comes to a list of links or something like but in cases like this is better to just use divs

File: maquetado/index.html [hunk 5]
```diff
@@ -8,27 +8,79 @@
...
+      <li>
+        <div class="book-cover">
+          <img class="book-image" src="image/book-cover.png" alt="Book Image" />
+          <div class="book-detail">
+            <h3 class="book-title">Título del libro</h3>
+            <p class="book-subtitle">Autor del libro</p>
+          </div>
+        </div>
+      </li>
```
Comment (Dorian30) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 6]
```diff
@@ -8,27 +8,79 @@
...
```
Comment (Dorian30) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 7]
```diff
@@ -8,27 +8,79 @@
...
```
Comment (Dorian30) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 8]
```diff
@@ -8,27 +8,79 @@
...
+      <li>
```
Comment (Dorian30) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 9]
```diff
@@ -8,27 +8,79 @@
...
+        <div class="book-cover">
+          <img class="book-image" src="image/book-cover.png" alt="Book Image" />
+          <div class="book-detail">
+            <h3 class="book-title">Título del libro</h3>
+            <p class="book-subtitle">Autor del libro</p>
+          </div>
+        </div>
+      </li>
+    </ol>
```
Comment (Dorian30) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 10]
```diff
@@ -8,27 +8,79 @@
...
+      </li>
+      <li>
+        <div class="book-cover">
+          <img class="book-image" src="image/book-cover.png" alt="Book Image" />
+          <div class="book-detail">
+            <h3 class="book-title">Título del libro</h3>
+            <p class="book-subtitle">Autor del libro</p>
+          </div>
+        </div>
+      </li>
```
Comment (Dorian30) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 11]
```diff
@@ -8,27 +8,79 @@
...
```
Comment (Dorian30) [inline maquetado/index.html]: same

File: maquetado/index.html [hunk 12]
```diff
@@ -8,27 +8,79 @@
...
-          <h3 class="book-detail-title">Editorial:</h3>
-          <p class="book-description">Nombre de la editorial</p>
-        </div>
-        <div class="book-detail">
-          <h3 class="book-detail-title">Año de publicación:</h3>
-          <p class="book-description">Año de publicación</p>
-        </div>
-      </div>
-    </div>
+    <ol class="list-of-books">
```
Comment (Dorian30) [inline maquetado/index.html]: Use a div instead ul and ol are used to wrapped small things like a list of links

File: maquetado/scss/detail.scss [hunk 13]
```diff
@@ -2,85 +2,42 @@
...
-  margin-left: 10px;
-  margin-top: 10px;
-  max-width: 1000px;
-  padding: 10px;
-  position: relative;
+  flex-direction: column;
+  height: 385px;
+  margin: 30px;
+  max-width: 265px;
+  padding: 30px 30px 50px 30px;
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: Use the shorthand

```suggestion
  padding: 30px 30px 50px;
```

**Review by Dorian30 — CHANGES_REQUESTED** [review]

**Review by Dorian30 — APPROVED** [review]

File: maquetado/scss/detail.scss [hunk 14]
```diff
@@ -2,89 +2,42 @@
...
-    content: "";
-    height: 5px;
-    margin-top: 20px;
-    position: absolute;
-    width: 640px;
-    top: calc(70%);
-  }
+  line-height: 20px;
+  margin-top: 10px;
+  margin-bottom: 10px;
```
Comment (gabrielizalo) [inline maquetado/scss/detail.scss:8]: Move `margin-bottom` before `margin-top`.

**Review by gabrielizalo — APPROVED** [review]

**Pull request merged**